Corona-Warn-App/src/main/java/de/rki/coronawarnapp/diagnosiskeys/server/AppConfigApiV1.kt

```diff
@@ -0,0 +1,11 @@
+package de.rki.coronawarnapp.diagnosiskeys.server
+
+import okhttp3.ResponseBody
+import retrofit2.http.GET
+import retrofit2.http.Path
+
+interface AppConfigApiV1 {
+
+    @GET("/version/v1/configuration/country/{country}/app_config")
```

Comment:
Disagree here, if there is ever a `v2` API, we should create a new `AppConfigApiV2` interface, exactly to prevent typo possibilities. Keep each separate, reduce side effects and uncomplicate testing. Thoughts? :thinking: